server/src/server.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
```

Comment:
Don't add these, if it is an actual linter problem try to fix it.

Reply:
Btw the fix for this is just giving the tournament code a type.
![image](https://github.com/user-attachments/assets/81f099ae-5e45-4609-8763-f45f0d453c8d)